src/components/RegisterSurveyComponent/RegisterSurveyComponent.jsx

```diff
@@ -46,243 +82,395 @@ class RegisterSurveyComponent extends Component {
+              </li>
+              <li>
+                <input
+                  className={styles.radioButton} 
+                  id="female"
+                  type="radio"
+                  name="gender"
+                  value="2"
+                  onClick={(e) => {
+                    this.setState({ gender_id: e.target.value });
+                  }}
+                />
+                <label htmlFor="female">Female</label>
+              </li>
+              <li>
+                <input
+                  className={styles.radioButton} 
+                  id='other'
+                  type="radio"
+                  name="gender"
+                  value="3"
+                  onClick={(e) => {
+                    this.setState({ gender_id: e.target.value });
+                  }}
+                />
+                <label htmlFor="other">Other</label>
+              </li>
+            </ul>
+            <button className={styles.registerButton} onClick={this.handleNext1}>
+              Next
+            </button>
+          </>
+        ) : ('')}
+
+        {this.state.activityLevelSelect ? (
+          <>
+            <h2>Activity Level</h2>
+            <li>
+              <input 
+                id="sedentary"
+                type="radio"
+                name="activityLevel"
+                value="1"
+                onClick={(e) => {
+                  this.setState({ activity_level_id: e.target.value });
+                }}
+              />
+              <label htmlFor="sedentary">Sedentary</label>
+            </li>
+            <li>
+              <input 
+                id="light"
+                type="radio"
+                name="activityLevel"
+                value="2"
+                onClick={(e) => {
+                  this.setState({ activity_level_id: e.target.value });
+                }}
+              />
+              <label htmlFor="light">Light</label>
+            </li>
+            <li>
+              <input
+                id='active'
+                type="radio"
+                name="activityLevel"
+                value="3"
+                onClick={(e) => {
+                  this.setState({ activity_level_id: e.target.value });
+                }}
+              />
+              <label htmlFor="active">Active</label>
+            </li>
+            <li>
+              <input 
+                id="veryActive"
+                type="radio"
+                name="activityLevel"
+                value="4"
+                onClick={(e) => {
+                  this.setState({ activity_level_id: e.target.value });
+                }}
+              />
+              <label htmlFor="veryActive">veryActive</label>
+            </li>
+            <button className={styles.registerButton} onClick={this.handleNext2}>
+              Next
+            </button>
+          </>
+        ) : ('')}
+
+        {this.state.goalSelect ? (
+          <>
+            <h2>Goal</h2>
+            <li>
```

Comment:
The list/button is getting cut off by the navbar in some views (galaxy / iphone). I think you need to put the goal list elements in a div and then apply flex to fix it so it's responsive.

Reply:
Flex box should be applied to all other list divs as well